spec/support/test_sidekiq_worker.rb

```diff
@@ -4,7 +4,12 @@ class TestSidekiqWorker
 
     def perform(options = {})
       return if options["success"]
-      raise "suck it nerd" if options["error"]
+      if options["error"]
+        raise "suck it nerd"
```

Comment:
I am a +1 on making this error less funny and more revealing. And since I'm probably the blame on this line, you should listen to me!